tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -204,6 +205,77 @@ test.describe.serial(`AI Lab extension installation and verification`, { tag: '@
     });
   });
 
+  ['instructlab/granite-7b-lab-GGUF'].forEach(modelName => {
+    test.describe.serial(`AI Lab playground creation and deletion`, () => {
+      let catalogPage: AILabCatalogPage;
+      let playgroundsPage: AILabPlaygroundsPage;
+
+      const playgroundName = 'test playground';
+
+      test.skip(isLinux, `Skipping model service creation on Linux`);
```

Comment:
just a typo after copying the message - model service.

Reply:
fixed